### DISCUSSION TITLE
How do I install Krita AI for portable version?

### DISCUSSION THREAD

**Discussion by nothingness6** [Acly/krita-ai-diffusion#488] [Q&A]
My C drive is running out, but I can’t find a solution of installing Krita AI on M drive that Krita portable is installed.
Github instruction only talks about installing on C:\Users\<user>\AppData\Roaming\krita\pykrita.
How do I do that? Is it even possible?

**Comment by MxtrSplinter** [comment]
The spot for the plugin and your server can be different, and you can change the install directory when doing the server convig through the plugin interface. I have Krita on my main drive, and the server install on another disk.

**Reply by nothingness6** [reply]
Hi, thanks for the reply. I installed Krita portable on D drive. So, I tried to install the plugin in the same folder, but the plugin doesn't show up in Python plugin manager. 
There was no pykrita folder, so I created it and put the plugin in it.

**Comment by Acly** [comment]
Where you install Krita doesn't matter. The plugin must be installed into Krita's resource folder. It is on C: by default, but can be relocated, please see Krita docs. But actually the plugin is very small. The server install is what requires all the disk space, and you can choose where to put it later from the plugin settings.

**Reply by nothingness6** [reply]
You're right. Thx!!
